pkg/sql/relocate_range.go

```diff
@@ -34,9 +34,9 @@ type relocateRange struct {
 // relocateRunState contains the run-time state of
 // relocateRange during local execution.
 type relocateRunState struct {
-	toStoreDesc   *roachpb.StoreDescriptor
-	fromStoreDesc *roachpb.StoreDescriptor
-	results       relocateResults
+	toTarget   roachpb.ReplicationTarget
```

Comment:
The whole `StoreDescriptor` is just converted to a `ReplicationTarget` later on so do it initially